### PR TITLE
Move import-beats script and package data

### PR DESCRIPTION
**Pull request by mtojek**
Issue: https://github.com/elastic/package-registry/issues/414

Done:
* Copy dev/import-beats script to the integrations repo
* Copy testing/environment script to the integrations and make it working (package registry in different repo)
* Add CI to integrations to still validate packages (copy from package-storage)

TODO:
*  update import-beats docs: README and CONTRIBUTING
* testing/environments: make it working

### CODE REVIEW

File: .ci/jenkins-go-agent/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM golang:1.14.2
```
Comment (ruflin) [inline .ci/jenkins-go-agent/Dockerfile:1]: @kuisathaverat Would be great to get a CI setup for this repo too, if we don't already have it.

File: .github/ISSUE_TEMPLATE/integration.md [hunk 2]
```diff
@@ -1,19 +0,0 @@
----
```
Comment (ruflin) [inline .github/ISSUE_TEMPLATE/integration.md:1]: Was this removed on purpose?

File: repository/README.md [hunk 3]
```diff
@@ -1,4 +0,0 @@
-# Integrations repository (ONLY EXPERIMENTAL USE)
```
Comment (ruflin) [inline repository/README.md:1]: Removed on purpose? Same for all the specs files?

File: .github/ISSUE_TEMPLATE/integration.md (hunk 2 above)
Comment (mtojek) [reply .github/ISSUE_TEMPLATE/integration.md:1]: This is an old file. Not relevant to what we'll have here.

File: repository/README.md (hunk 3 above)
Comment (mtojek) [reply repository/README.md:1]: Yes. All these files contain nothing relevant. Only placeholders.

Comment (ruflin) [reply repository/README.md:1]: I see. @andresrc FYI

File: .ci/jenkins-go-agent/Dockerfile (hunk 1 above)
Comment (mtojek) [reply .ci/jenkins-go-agent/Dockerfile:1]: If this is a matter of adjusting Jenkinsfile, I can handle it in the followups.

**Comment by mtojek** [comment]
> To make this PR better reviewable, perhaps first just push in all the automation and scripts as part of the PR and remove the packages itself?

Done.

**Comment by mtojek** [comment]
@ruflin I'm ok with reviewing and pushing the current state of PR and then apply next changes in followups (testing/environment, CI, docs).

File: .ci/jenkins-go-agent/Dockerfile (hunk 1 above)
Comment (ruflin) [reply .ci/jenkins-go-agent/Dockerfile:1]: It is about adding this repo to JJBB, needs to happen in an other repo.

File: magefile.go [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+	}
+
+	for _, p := range packagePaths {
+		err := sh.Run("go", "run", "github.com/elastic/package-registry/dev/generator/", "-sourceDir="+p, "-publicDir="+publicDir, "-tarGz="+strconv.FormatBool(tarGz))
+		if err != nil {
+			return err
+		}
+	}
+
+	err = BuildRootFile()
```
Comment (ruflin) [inline magefile.go:58]: This will probably not be needed anymore in here?

Comment (mtojek) [reply magefile.go:58]: Not sure yet. I want to build the package directory with gzipped artifacts and mount it in the container image with EPR. I'll come back to this.

**Pull request merged**